gateway/src/test/java/io/zeebe/gateway/broker/BrokerClientTest.java

```diff
@@ -417,93 +298,11 @@ public void shouldReturnRejectionWithCorrectTypeAndReason() {
     assertThat(rejection.getReason()).isEqualTo("foo");
   }
 
-  @Test
-  public void shouldFailRequestIfTopologyCannotBeRefreshed() {
-    // given
-    broker.onTopologyRequest().doNotRespond();
-    broker.onExecuteCommandRequest(ValueType.JOB, JobIntent.COMPLETE).doNotRespond();
-
-    // then
-    exception.expect(ExecutionException.class);
-    exception.expectMessage("Request timed out after PT1S");
-
-    // when
-    client.sendRequest(new BrokerCompleteJobRequest(0, EMPTY_PAYLOAD)).join();
-  }
-
-  @Test
-  public void shouldRetryTopologyRequestAfterTimeout() {
```

Comment:
Do we still have a test which checks that the request is retried with an updated topology after initial request failure, i.e. if a topology change happend while the broker request was inflight. If there is no such test case we should readd one.